### PR TITLE
[MetricsAdvisor] Added README and samples

### PR DESCRIPTION
**Pull request by kinelski**

### CODE REVIEW

File: sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md [hunk 1]
```diff
@@ -1,45 +1,466 @@
...
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][code_of_conduct]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
 ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fmetricsadvisor%2FAzure.AI.MetricsAdvisor%2FREADME.png)
+
+<!-- LINKS -->
+[metricsadv_client_src]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src
+<!--[metricsadv_docs]: https://docs.microsoft.com/en-us/azure/cognitive-services/metrics-advisor
+[metricsadv_nuget_package]: https://www.nuget.org/packages/Azure.AI.MetricsAdvisor-->
```
Comment (christothes) [inline sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md:445]: We can use this as a placeholder until the package releases:
`https://www.nuget.org/packages?q=Azure.AI.MetricsAdvisor`

File: sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md [hunk 2]
```diff
@@ -1,45 +1,466 @@
...
-Forthcoming
+Azure Cognitive Services Metrics Advisor is a cloud service that uses machine learning to monitor and detect anomalies in time series data. It includes the following capabilities:
 
-## Getting started
+- Analyze multi-dimensional data from multiple data sources.
+- Identify and correlate anomalies.
+- Configure and fine-tune the anomaly detection model used on your data.
+- Diagnose anomalies and help with root cause analysis.
+
+[Source code][metricsadv_client_src] | Package (NuGet) | [API reference documentation][metricsadv_refdocs] | Product documentation | [Samples][metricsadv_samples]
```
Comment (christothes) [inline sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md:10]: Let's add the link to `metricsadv_nuget_package` here.

Comment (christothes) [reply sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md:10]: Link product docs to here https://docs.microsoft.com/en-us/azure/cognitive-services/metrics-advisor/

File: sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md [hunk 3]
```diff
@@ -1,45 +1,466 @@
...
+
+[metricsadv-sample1]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample1_CreateDataFeedFromDataSource.cs
+[metricsadv-sample2]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample2_CheckIngestionStatusOfDataFeed.cs
+[metricsadv-sample3]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample3_CreateAnomalyDetectionConfiguration.cs
+[metricsadv-sample4]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample4_CreateHookForReceivingAnomalyAlerts.cs
+[metricsadv-sample5]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample5_CreateAnomalyAlertConfiguration.cs
+[metricsadv-sample6]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample6_QueryDetectedAnomaliesAndTriggeredAlerts.cs
```
Comment (christothes) [inline sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md:459]: Did you intend to make these links to your fork?

**Review by christothes — COMMENTED** [review]
Left a few comments. Looks good overall.

File: sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample1_CreateDataFeedFromDataSource.cs [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+            {
+                new MetricDimension("category"),
+                new MetricDimension("city")
+            };
+            var dataFeedSchema = new DataFeedSchema(dataFeedMetrics)
+            {
+                DimensionColumns = dataFeedDimensions
+            };
+
+            var ingestionStartTime = DateTimeOffset.Parse("2020-01-01T00:00:00Z");
```
[the remark this answers is not in the document]
Comment (kinelski) [reply sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample1_CreateDataFeedFromDataSource.cs:52]: I wanted to make clear to readers what this time meant.

File: sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md (hunk 3 above)
Comment (kinelski) [reply sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md:459]: These files don't exist in master yet. I need to create another PR after this one to update them.

File: sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md (hunk 2 above)
Comment (kinelski) [reply sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md]: Just got a bit confused. `API reference documentation` was the one I should have removed.

Comment (kinelski) [reply sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md]: Problem was the `en-us` bit. Fixed.

**Review by christothes — APPROVED** [review]

**Pull request merged**